<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportCCB</class>
 <widget class="QDialog" name="DialogImportCCB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathCCB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CCB File Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseCCB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>46</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathResources">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseResources">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>106</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resources Path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogImportCCB</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImportCCB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
